NOTICE

```diff
@@ -19,4 +19,4 @@ organizations and licensed in different ways:
 * `tools`
 Please refer to their LICENCE, COPYING or NOTICE files for terms of use.
 Some icons files may be copyrighted by (C) 2020 My.com B.V. (Mail.Ru Group)
-See also `data/copyright.html` file for a full list of copyright notices.
+See also the `.reuse/dep5` and `data/copyright.html` file for a full list of copyright notices.
```

Comment:
So, now we have 2 files where we need to update licenses. Doesn't look convenient for me.
What can we do about this?